src/citrine/resources/data_concepts.py

```diff
@@ -503,18 +503,26 @@ def register_all(self,
                      dry_run=False,
                      status_bar=False) -> List[ResourceType]:
         """
-        [ALPHA] Create or update each model in models.
+        Register multiple GEMD objects to each of their appropriate collections.
 
-        This method has the same behavior as `register`, except that all no models will be
-        written if any one of them is invalid.
+        Does so in an order that is guaranteed to store all linked items before the item that
+        references them.
+
+        If the GEMD objects have no UIDs, Citrine IDs will be assigned to them prior to passing
+        them on to the server.  This is required as otherwise there is no way to determine how
+        objects are related to each other.  When the registered objects are returned from the
+        server, the input GEMD objects will be updated with whichever uids & _citr_auto:: tags are
+        on the returned objects.  This means GEMD objects that already exist on the server will
+        be updated with all their on-platform uids and tags.
 
-        Using this method should yield significant improvements to write speed over separate
-        calls to `register`.
+        This method has the same behavior as `register`, except that all no models will be
+        written if any one of them is invalid.  Using this method should yield significant
```

Comment:
Is this supposed to say "except that no models will be written if..."?

Reply:
Good catch.